cfme/networks/provider/nuage.py

```diff
@@ -77,6 +81,10 @@ class NuageProvider(NetworkProvider):
     def __attrs_post_init__(self):
         self.parent = self.appliance.collections.network_providers
 
+    @property
+    def cloud_tenants(self):
+        return self.collections.cloud_tenants
```

Comment:
I think that this qualifies as a 'convenience method', and should not be included. The caller should reference entity collections directly through BaseEntity.collections, this property provides no additional functionality.